### PR TITLE
Deprecate console output in favor of JSDebugger

### PR DESCRIPTION
**Pull request by bgoddar**
This PR updates our Console output to inform users that it will be deprecated and direct folks to use JSDebugger instead.

### CODE REVIEW

File: src/devtoolsPanel.ts [hunk 1]
```diff
@@ -54,18 +54,13 @@ export class DevToolsPanel {
...
-            this.consoleOutput.appendLine('// Note that this feature is only unidirectional and cannot communicate back to the DevTools.');
-            this.consoleOutput.appendLine('');
-        }
+
+        // Deprecated console
+        // Direct users to the Debug Console. This message with be removed in a the next update
+        this.consoleOutput.appendLine('// Microsoft Edge Devtools Extension:');
+        this.consoleOutput.appendLine('// The Microsoft Edge DevTools Extension will be deprecating console output in the next update.');
+        this.consoleOutput.appendLine('// To receive full REPL console functionality, please use Visual Studio Code\'s included JavaScript Debugger to attach to your target.');
+        this.consoleOutput.appendLine('// Then use the "Debug Console" to view console messages from your webpage and evaluate expressions.');
```
Comment (bgoddar) [inline src/devtoolsPanel.ts:63]: @codepo8  is there a doc link I can add here with more info? I couldn't find a good one for JSDebug console offhand

Comment (codepo8) [reply src/devtoolsPanel.ts:63]: This is part of the official docs for the extension in the debugging section:
https://docs.microsoft.com/microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension#browser-debugging-with-microsoft-edge-developer-tools-integration-in-visual-studio-code

Comment (codepo8) [reply src/devtoolsPanel.ts:63]: Once we added the link, this LGTM.

**Review by codepo8 — COMMENTED** [review]
Text looks good to me, and if we can add the link, then we're ready to go.

**Review by flynnolivia — APPROVED** [review]
LGTM!

**Pull request merged**